app/policies/comment_policy.rb

```diff
@@ -0,0 +1,24 @@
+class CommentPolicy < ApplicationPolicy
+  attr_reader :user, :comment
+
+  def initialize(user, comment)
+    @user = user
+    @comment = comment
+  end
+
+  def show?
+    true
+  end
+
+  def edit?
+    comment.author_id == user.id
+  end
+
+  def update?
+    comment.author_id == user.id
+  end
+
+  def destroy?
+    user == comment.author
```

Comment:
I think the edit and update policies could be the same as this one and it's a bit more readable -- but that's not a necessary change